docs/sources/writing-guide/shortcodes/index.md

```diff
@@ -143,6 +143,43 @@ This shortcode inserts a lists of links to the page's subpages and includes the
 {{</* section withDescriptions="true"*/>}}
 ```
 
+## `docs/reference` shortcode
+
+The `docs/reference` shortcode offers more flexible linking than the Hugo builtin `relref` shortcode.
+Reference links wrapped in the `docs/reference` shortcode can be defined with multiple destinations.
+The destination used in a the rendered page depends is chosen based upon matching path prefixes.
+
+```markdown
+{{%/* docs/reference */%}}
+[<LABEL>]: "<PATH PREFIX> -> <REFERENCE>"
+...
+{{%/* /docs/reference */%}}
+```
+
+- _`LABEL`_ matches a link label used in a reference-style link (`[LINK TEXT][LABEL])`).
+  For more information about reference-style links, refer to [Daring Fireball: Markdown Syntax Documentation](https://daringfireball.net/projects/markdown/syntax#link).
+- _`PROJECT PATH PREFIX`_ is matched against the page’s URL path.
```

Comment:
```suggestion
- _`PROJECT PATH PREFIX`_ is matches the page’s URL path.
```

Reply:
I feel "is matched against" is more clear because there is a selection process where only the first matching _`PROJECT PATH PREFIX`_ is chosen.